.env.erb

```diff
@@ -1,5 +1,5 @@
-DB_USERNAME=username_without_quotes
-DB_PASSWORD=password_without_quotes
+DB_USERNAME=postgres
+DB_PASSWORD=Pas4all@1
```

Comment:
We use this file as a template for new developers. that is why it's .env.erb, so when you add the secret's for google, bing or brave API the developer can just remove the ".erb" part and reutilize it.